src/utils/layout.js

```diff
@@ -31,6 +31,11 @@ function isPageBreakSection(section) {
       section.data && ((isString(section.data) ? section.data : section.data.text) || '').includes(PAGE_BREAK_KEY));
 }
 
+export function shouldSectionBeNotBordered(section) {
```

Comment:
Is it possible to use css rules instead of this function (and the isBordered prop) to achieve the desired border style?

Reply:
probably yes... but with our time limits... 🙅  🙈